lib/repack.c

```diff
@@ -1189,6 +1189,7 @@ swap_heap_or_index_files(Oid r1, Oid r2)
 	Form_pg_class relform1,
 				relform2;
 	Oid			swaptemp;
+	TransactionId swaptempxid;
```

Comment:
`swaptempxid` can be declared inside of the `if (relform1->relkind != RELKIND_INDEX)` block.